engine/samples/events/main.cpp

```diff
@@ -5,71 +5,85 @@
 
 using namespace cubos::core;
```

Comment:
I think replacing this with `using cubos::core::ecs::Blabla` for each used type would be more useful.
Then you could also show in the sample docs that `using cubos::core::ecs::EventReader/EventWriter` is being used for convenience.